feature/feature.go

```diff
@@ -86,6 +86,16 @@ const (
 	// owner: @enxebre
 	// alpha: v2.2
 	ROSA featuregate.Feature = "ROSA"
+
+	// AWSMachine is used to control AWSMachine reconciler
+	// owner: @mzazrivec
+	// beta: v2.7
+	AWSMachine featuregate.Feature = "AWSMachine"
```

Comment:
With hindsight i think these should be a new single flag to disable both and not feature flags.

Reply:
In the proposal we discussed that theoretically end user can use awsCluster with machinePool without using awsMachine. That was the main reason for having 2 feature gates.

Reply:
"awsCluster with machinePool without using awsMachine" that's not a goal of CAPA. With CAPA (excluding Rosa) the only choice is around enabling feature. This scenario is not something we want to support at this stage.

Reply:
that is okay, its just during discussion we ask is that possible and answer was could be :)